### PR TITLE
board story demonstrates issue with margin collapse + placeholder

### PR DESCRIPTION
**Pull request by jaredcrowe**
just creating this PR to get the storybook build, feel free to decline it

### CODE REVIEW

File: stories/src/primatives/quote-list.jsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ const Wrapper = styled.div`
   display: flex;
   flex-direction: column;
   opacity: ${({ isDropDisabled }) => (isDropDisabled ? 0.5 : 'inherit')};
-  padding: ${grid}px;
```
Comment (alexreardon) [inline stories/src/primatives/quote-list.jsx]: I am not sure if it is feasible to detect for margin collapsing for droppable placeholders. My current thinking is to add a note about it in the docs and leave it at that. Thoughts?

Comment (jaredcrowe) [reply stories/src/primatives/quote-list.jsx]: might have to be. I'm spiking a solution atm but I'm not sure if it'll cover everything. I'll report back 👨‍🚀

**Comment by alexreardon** [comment]
Can we close this one?

**Comment by jaredcrowe** [comment]
yup 👍

**Pull request closed without merging**